src/components/VolunteerForm/Fields/Input.js

```diff
@@ -57,73 +59,77 @@ export const ConfirmMobileField = () => (
       }),
     ]}
   >
-    <Input maxLength={10} placeholder="Enter re-enter your mobile number" />
+    <Input maxLength={10} placeholder={t("reenter_number")} />
   </Form.Item>
-);
+)};
 
-export const AddressField = () => (
-  <Form.Item label="Address" name="address">
+export const AddressField = () => {
+  const [t] = useTranslation();
+  return(
+  <Form.Item label={ t("address") } name="address">
     <TextArea
       maxLength={100}
       type="textarea"
       rows={4}
-      placeholder="Enter your address"
+      placeholder={t("placeholder.enter_address")}
     />
   </Form.Item>
-);
+)};
 
-export const PinField = () => (
+export const PinField = () => {
+  const [t] = useTranslation();
+  return(
   <Form.Item
-    label="Pin Code"
+    label= {t("pin_code")}
     name="pin"
     rules={[
       {
         required: true,
-        message: "Pin code is required",
+        message: t("validation_messages.pin_code_required"),
       },
       {
         type: "number",
         transform: toNumber,
-        message: "Invalid format",
+        message: t("validation_messages.pin_code_invalid_format"),
       },
       {
         len: 6,
-        message: "Pin must be 6 digits",
+        message: t("validation_messages.pin_code_length"),
       },
     ]}
   >
-    <Input maxLength={6} placeholder="Enter pin code" />
+    <Input maxLength={6} placeholder={t("placeholder.pin_code")} />
   </Form.Item>
-);
+)};
 
-export const NameField = ({
-  label = "Name",
-  placeholder = "Enter your full name",
-  name = "name",
-}) => (
+export const NameField = () => {
```

Comment:
you will have to rebase and update this section, I had to parameterize name filed for some new use cases.

Reply:
I have rebased the branch, but the parameters of label ,placeholder i have removed

Reply:
Not sure I understand, we need the parameters for mobile as it's used under different name/labels.

Reply:
if it is going to be dynamic then i would revert back it, is it fine ?